packages/web/components/view-mode-renderer.tsx

```
@@ -227,18 +231,18 @@ export function ViewModeRenderer({ note, className }: ViewModeRendererProps) {
             </header>
 
             {/* Content */}
-            <div className="prose prose-lg max-w-none">
+            <div className='prose prose-lg max-w-none'>
               {note.content ? (
                 <div
-                  className="view-mode-content"
+                  className='view-mode-content'
                   dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)